tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -923,3 +923,62 @@ def test_optuna_callback(self) -> None:
             tuner._tune_params(["num_leaves"], 10, optuna.samplers.TPESampler(), "num_leaves")
 
         assert callback_mock.call_count == 10
+
+    def test_get_best_booster(self) -> None:
+        unexpected_value = 20  # out of scope.
+
+        params = {"verbose": -1, "lambda_l1": unexpected_value}  # type: Dict
+        dataset = lgb.Dataset(np.zeros((10, 10)))
+        study = optuna.create_study()
+
+        with TemporaryDirectory() as tmpdir:
+            tuner = LightGBMTunerCV(
+                params, dataset, study=study, model_dir=tmpdir, return_cvbooster=True
+            )
+
+            with pytest.raises(ValueError):
+                tuner.get_best_booster()
+
+            with mock.patch.object(_OptunaObjectiveCV, "_get_cv_scores", return_value=[1.0]):
+                tuner.tune_regularization_factors()
+
+            best_boosters = tuner.get_best_booster().boosters
+            for booster in best_boosters:
+                assert booster.params["lambda_l1"] != unexpected_value
+
+                tuner2 = LightGBMTunerCV(
+                    params, dataset, study=study, model_dir=tmpdir, return_cvbooster=True
+                )
+                best_boosters2 = tuner2.get_best_booster().boosters
+                for booster, booster2 in zip(best_boosters, best_boosters2):
+                    assert booster.params == booster2.params
```

Comment:
I think these lines are over-indented.
```suggestion
            tuner2 = LightGBMTunerCV(
                params, dataset, study=study, model_dir=tmpdir, return_cvbooster=True
            )
            best_boosters2 = tuner2.get_best_booster().boosters
            for booster, booster2 in zip(best_boosters, best_boosters2):
                assert booster.params == booster2.params
```

Reply:
Thank you for pointing out, fixed those lines.